### ISSUE TITLE
What are the different ways to visually hide content (and make it available only for screen readers)?

### ISSUE THREAD

**Issue by martinjunior** [yangshun/front-end-interview-handbook#241]
The answer to this question lists `visibility: hidden`. `visibility: hidden;` hides content from screen readers, so it should be excluded from the list.
